meetups/brno.yml

```
@@ -477,3 +477,13 @@
   date: 2023-06-10
   time: 9:00 – 13:00
   place: *fi-muni
+
+  - name: Začátečnický kurz naruby
```

Tahle odrážka by neměla být odsazená (viz předchozí záznam)

```suggestion
- name: Začátečnický kurz naruby
```